client/src/app/components/research-project-card/research-project-card.component.ts

```diff
@@ -58,7 +58,7 @@ export class ResearchProjectCardComponent implements OnInit {
   }
 
   redirectToCreateProject() { 
-    this.router.navigate(['/create-post']);
+    this.router.navigate(['/create-post/Active/new']);
```

Comment:
is there a way to make active lowercase?

Reply:
no, its a URL parameter and not an end point so its value matters

Reply:
gotcha. I guess i don't know if its best practice or not to keep front-end routes lowercase. Stherwise, resolve those conflicts, and LGTM!